Kontent.Ai.Management/Models/Spaces/SpaceResponseModel.cs

```diff
@@ -0,0 +1,28 @@
+using Newtonsoft.Json;
+using System;
+
+namespace Kontent.Ai.Management.Models.Spaces;
+
+/// <summary>
+/// Represents the space response model.
+/// </summary>
+public class SpaceResponseModel
```

Comment:
SDK does not use Response suffix -> `SpaceModel`

Reply:
Renamed all occurrences to SpaceModel